content/updates/2020-09/1-flat-files-exports-with-labels.md

```diff
@@ -0,0 +1,31 @@
+---
+pim_announcement_img: ./img/Product-export-profile.png
+pim_announcement_alt_img: "Flat files exports with labels"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Export flat files with labels
+::: meta-data type="New" features="Collaboration, Productivity" available="mid September" link-to-doc="../articles/exports.html"
+:::
+
+To save you time, it is now possible to export your product data in flat files with labels and without codes.
```

Comment:
If you want to go to the next line, you'll need to add two spaces at the end of your line.
```suggestion
To save you time, it is now possible to export your product data in flat files with labels and without codes.  
```